### ISSUE TITLE
feature request: `split_search` by % identities

### ISSUE THREAD

**Issue by tkosciol** [biocore/microprot#72]
we have an option to request `max_pvalue`, `max_evalue` and `min_prob`. It would be nice to also have an option to request minimum pair-wise sequence identity (`min_ident`) which can be directly read from HHsearch output, e.g.:
```
No 1
>1FP3_B N-ACYL-D-GLUCOSAMINE 2-EPIMERASE FROM PORCINE KIDNEY; ALPHA/ALPHA-BARREL, N-acyl-D-glucosamine 2-epimerase, ISOMERASE; 2.0A {Sus scrofa} SCOP: a.102.1.3
Probab=96.16  E-value=0.00015  Score=60.96  Aligned_cols=290  **Identities=16%**  Similarity=0.102  Sum_probs=145.1  Template_Neff=10.900
```

**Comment by tkosciol** [comment]
identities highlighted by "**" in quote above

**Comment by sjanssen2** [comment]
let me work on that as soon as we solved #71

**Comment by sjanssen2** [comment]
Good that I have foreseen that sometimes someone might need to operate on those information that are only accessable in the alignments and not in the leading table. Thus, this feature request should be easy. PR #74 should address it.
